lib/src/author/build.rs

```diff
@@ -229,6 +229,7 @@ impl AuthoringStart {
                     })
                 }
             },
+            max_log_level: Default::default(),
```

Comment:
This should be a field in the `Config` struct found in `build.rs`

Reply:
thought we don't need it in this case, adding it